src/login/Login.jsx

```diff
@@ -31,7 +31,7 @@ export default function Login() {
                 if (response.status === 200) {
                     login(data, user);
                 }
-                setErrorMessage(data["message"]);
+                setErrorMessage(data.message);
             })
             .catch(() => setErrorMessage("The server is currently unavailable. Try again later"));
```

Comment:
Can this error message be a constant and just imported everywhere it's used?

Reply:
Done. It's now a constant 😁